product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -279,14 +410,27 @@ To restore a deleted cluster, use the `--from-deleted` flag in the command.
 You can restore a cluster in a single cluster to a high-availability cluster and vice versa. You can restore an extreme-high-availability cluster  only to a cluster using the same architecture.
 !!!
 
-## Using `pgd` commands
 
-Use the `pgd` commands to create, retrieve information on, and manage extreme-high-availability clusters. 
+## Managing extreme-high-availability clusters
+
+Use the BigAnimal `pgd` commands to create, retrieve information on, and manage extreme-high-availability clusters. 
 
+In addition to these BigAnimal `pgd` CLI commands, you can use [EDB Postgres Distributed CLI  commands](/pgd/latest/cli/) to manage your extreme-high-availability clusters. 
 
-### `pgd create` 
+!!!warning
```

Comment:
This entire procedure is appearing within the **Warning** box. Is that intentional?

Reply:
nope, not intentional. thanks for checking that